operator/src/main/java/oracle/kubernetes/operator/helpers/PodStepContext.java

```diff
@@ -789,6 +803,7 @@ void overrideContainerWeblogicEnvVars(List<V1EnvVar> vars) {
     addEnvVar(vars, "DOMAIN_HOME", getDomainHome());
     addEnvVar(vars, "ADMIN_NAME", getAsName());
     addEnvVar(vars, "ADMIN_PORT", getAsPort().toString());
+    addEnvVar(vars, "ADMIN_PORT_SECURE", Boolean.toString(isLocalAdminProtocolChannelSecure()));
```

Comment:
@alai8 This will cause existing pods to roll during an operator upgrade (correct, @russgold?) Assuming that's correct, I would suggest that you only add this environment variable if the value is "true" and then account for the variable not being set inside the pod.